mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/DependencyResolver.java

```diff
@@ -60,8 +60,8 @@ private Dependency smartTestingProviderDependency() {
     }
 
     static class SurefireApiDependency extends Dependency {
-        private static final String GROUP_ID = "org.apache.maven.surefire";
-        private static final String ARTIFACT_ID = "surefire-api";
+        static final String GROUP_ID = "org.apache.maven.surefire";
+        static final String ARTIFACT_ID = "surefire-api";
```

Comment:
nope - it's zombie from my previous version - I realized when I left the office, but it was too late :-)